bark/python_wrapper/world/agent.cpp

```diff
@@ -71,10 +73,11 @@ void python_agent(py::module m) {
                 a.GetExecutionModel(),                           // 7
                 a.GetDynamicModel(),                             // 8
                 a.GetCurrentState(),                             // 9
-                GoalDefinitionToPython(a.GetGoalDefinition()));  // 10
+                a.GetFirstValidTimestamp(),                      // 10
```

Comment:
Should this not be handled by the behavior model?

Reply:
wasn't sufficient. We might consider removing the BehaviorStatus